.rubocop.yml

```diff
@@ -0,0 +1,37 @@
+require: 
+  - rubocop-rails
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - 'db/**/*'
+    - 'bin/**/*'
+    - 'Gemfile'
+    - 'Rakefile'
+    - 'spec/spec_helper.rb'
+    - 'node_modules/**/*'
+    - 'tmp/*'
+    - 'config/initializers/**/*'
+    - 'lib/tasks/*'
+    - 'Guardfile'
+
+Layout/LineLength:
+  Max: 100
+  Exclude: 
+    - 'config/routes.rb'
+
+Metrics/BlockLength:
+  Exclude:
+    - config/**/*
+    - spec/**/*
```

Comment:
I wonder if using the [ExcludedMethods attribute](https://docs.rubocop.org/rubocop/1.0/cops_metrics.html#configurable-attributes-2) of this cop would give us more fidelity? If so, i'd like to keep `config` and `spec` in the fold.

Reply:
no problem